src/Go.Windows/Logger.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace Go.Windows
+{
+    using System;
+    using System.ComponentModel.Composition;
+    using System.Text;
+    using Microsoft.VisualStudio;
+    using Microsoft.VisualStudio.Shell;
+    using Microsoft.VisualStudio.Shell.Interop;
+
+    [Export]
+    internal sealed class Logger
```

Comment:
Can we add an ILogger interface with this same set of methods to Go.Editor? This will be used by the Editor layer to write to the VS Windows or VS Mac output windows respectively.